qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -25,6 +25,7 @@ class Env {
     static final CI_TAG = System.getenv("CIRCLE_TAG")
     static final GATHER_QA_TEST_DEBUG_LOGS = (System.getenv("GATHER_QA_TEST_DEBUG_LOGS") == "true")
     static final QA_TEST_DEBUG_LOGS = System.getenv("QA_TEST_DEBUG_LOGS") ?: ""
+    static final HW_ARCH = System.getenv("CLUSTER_HW_ARCH")
```

Comment:
It will be easier for future readers if you specify that this variable is the remote cluster architecture, especially since, as of now, the central arch is `x86_64`, but the remote cluster arch can be `ppc64le` or  `s390x`. Please change the groovy and environment variable names to be more specific, e.g.,`REMOTE_CLUSTER_ARCH.`

Reply:
changes done